src/components/atoms/Input/helpers/TextInput.tsx

```diff
@@ -116,6 +112,23 @@ export const TextInput: TextInput = ({
     else setLocalValue(value)
   }, [register, setValue, name, value])
 
+  // eslint-disable-next-line consistent-return
+  React.useEffect((): (() => void) | void => {
+    const button = document.querySelector('.show-password')
+    const input: HTMLInputElement = document.querySelector(`#${name}`) as HTMLInputElement
```

Comment:
No need for `querySelector` here:

```suggestion
    const input: HTMLInputElement = document.getElementById(name) as HTMLInputElement
```